packages/gatsby/src/commands/develop.js

```diff
@@ -125,7 +125,7 @@ async function startServer(program) {
 
   if (process.env.GATSBY_GRAPHQL_IDE === `playground`) {
     app.get(
-      `/___graphql`,
+      `/_+graphi{0,1}ql`,
```

Comment:
Nice! 

Small nitpick:
What about extracting this to const variable that can be used in both route handlers and add some context in comment before that:
```
/**
 * Pattern matching all endpoints with graphql or graphiql with 1 or more leading underscores
 */
const graphqlEndpoint = `/_+graphi{0,1}ql
```

Reply:
That sounds like a good idea, thanks!